proxy/frontend_handler.go

```diff
@@ -53,8 +62,22 @@ func (h *FrontendHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 			}
 			switch message.Type {
 			case common.Body:
+				var data []byte
+				var e error
+				msgType := 1
+				if binary {
+					msgType = 2
+					data, e = base64.StdEncoding.DecodeString(message.Body)
+					if e != nil {
+						log.Errorf("Error decoding message: %v", e)
+						closeConnection(ws)
```

Comment:
It wont blow up, but the ws.WriteMessage will return an error and it will attempt to close the connection again.
So, I'll add a continue here. It cannot be a break because the only thing that is allowed to close this go routine is the channel closing (otherwise, the multiplexer will block because there is no receiver for the channel.